### DISCUSSION TITLE
Is there a web UI to this? Port?

### DISCUSSION THREAD

**Discussion by kmanan** [dickwolff/Export-To-Ghostfolio#74] [Q&A]
Hi. n00b here, just trying to figure out how to use this to import stuff to ghostfolio, how do I access this tool to convert the file? I am using TrueNAS Scale and tried to pull this container, unsure if this uses a web UI / if yes, is there an port I should ocnfigure?

**Comment by dickwolff** [comment]
Hi there! There is no web UI (yet), and you dont need to map any ports to run the container either. 

You have to set a couple of [environment variables](https://github.com/dickwolff/Export-To-Ghostfolio?tab=readme-ov-file#how-to-use-with-the-dockerhub-image) (see `How to use with the Dockerhub image`) and then you should be able to use it. I suggest to set polling as well, this will keep the container running.

Basically you mount some folders to the container and then you can put a broker export file in the input folder and the container will put the converted file in the output folder.

Should you have any other questions, let me know!
